arm-eventhub/2015-08-01/swagger/EventHub.json

```diff
@@ -0,0 +1,1771 @@
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "namespaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "name": "eventHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The EventHub name."
+          },
+          {
+            "name": "consumerGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Consumer Group name."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConsumerGroupCreateOrUpdateParameters"
+            },
+            "description": "Parameters supplied to create a Consumer Group Resource."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Consumer Group created.",
+            "schema": {
+              "$ref": "#/definitions/ConsumerGroupResource"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ConsumerGroups"
+        ],
+        "operationId": "ConsumerGroups_Delete",
+        "description": "Deletes an ConsumerGroup from the specified EventHub and resource group.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639491.aspx"
+        },
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "namespaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "name": "eventHubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The EventHub name."
+          },
+          {
+            "name": "consumerGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Consumer Group name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Consumer Group deleted."
+          }
```

Comment:
- should this have a 202?
- the create is not long running but the delete is. Is that correct?